sdk/servicebus/service-bus/package.json

```diff
@@ -73,7 +73,7 @@
     "test:node": "npm run clean && npm run build:test:node && npm run integration-test:node",
     "test": "npm run test:node && npm run test:browser",
     "unit-test:browser": "echo skipped",
-    "unit-test:node": "mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 120000 --full-trace \"dist-esm/test/internal/unit/*.spec.js\" \"dist-esm/test/internal/node/*.spec.js\"",
+    "unit-test:node": "nyc --report-dir coverage/unit-test mocha -r esm -r ts-node/register --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 120000 --full-trace \"test/internal/unit/*.spec.ts\" \"test/internal/node/*.spec.ts\"",
```

Comment:
Any reason to use nyc here? Wasnt the prior decision to only run code coverage in the `integration` script that gets run nightly and not the `unit-test` script that gets run on every PR to reduce the time taken for getting code coverage?

Reply:
Hm...I didn't realize that was an official decision (and not just an omission).

The coverage output is human readable (ie, it's HTML) and it's extremely useful when you're trying to get a good picture of how you're covering existing code. I'd say leave it unless you think it's going to cause harm (and we should encourage people to add it in in their unit-test targets).

Reply:
Yes, no need of nyc here.

Reply:
Majority wins - I will remove. :)